CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 3.0.4 (TBD)
-* Allow for specifying a comma seperated string instead of array for the `services` and `formats` filters.
+* The `services` and `formats` filter parameters for the `GetSearchResults` API now accept a comma separated string (e.g. `services=1,-12,42`) in addition to the standard array syntax.
```

Comment:
this actually works for GetServiceBodies with services filter as well as GetFormats with formats filter which is why i didnt explicitly only call out GetSearchResults. but this is fine, this is what most users want to know